src/Browsershot.php

```diff
@@ -645,6 +674,17 @@ protected function getNodePathCommand(string $nodeBinary): string
         return 'NODE_PATH=`npm root -g`';
     }
 
+    protected function getOptionsCommand(string $command): string
```

Comment:
Do we know when a command becomes too large? Could we automatically switch to the options file whenever it becomes too large?

Reply:
I don't think we can use this as an automatic fallback. We've noticed in other packages that some systems don't properly support temporary directories (I believe there were some issues in our image package with it). I think it could break people's projects if we automatically switch to using this temporary directory approach. I could be wrong, but I don't think it's worth the risk.

Reply:
Ok, let's leave this as it is in this PR for now.